packages/cli/README.md

```diff
@@ -992,7 +1010,7 @@ DESCRIPTION
   create a capability translation
 
 EXAMPLES
-  $ smartthings capabilities:translations:create custom1.outputModulation 1 -i en.yaml 
```

Comment:
I usually run `npm run readme` (or `npm run build`) after making changes to the README to remove this minor noise.